### ISSUE TITLE
reload option in main file menu

### ISSUE THREAD

**Issue by ORESoftware** [Microsoft/vscode#62529]
It sort of baffles me why this is not already available in the main menu:

![screenshot from 2018-11-03 20-03-33](https://user-images.githubusercontent.com/11139560/47959526-98bd3480-dfa3-11e8-9a25-080aaffdf832.png)

in the file menu, I have to select exit and then manually re-open vscode.

in the file menu there should be a "reload" option in the dropdown so that I can restart vscode upon some setting change without having to manually open the app after it's closed/exited.

Basically what I want is the "reload" button in the screenshot available from the file menu, thx.

**Comment by avannaldas** [comment]
I can take this up, looking into it.

**Comment by ORESoftware** [comment]
thx

**Comment by isidorn** [comment]
Use `F1 > reload window`

We will not add it to the file menu, it provides clutter. Also the reload command has nothing to do with files and does not belong in that menu at all.
Closing as designed
